### DISCUSSION TITLE
Exploring to sickcodes/Docker-OSX runs device out of memory

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44771] [Mobile] [answered]
### Select Topic Area

Bug

### Body

GitHub Mobile beta
(Google Pixel 3A with Google podcasts playing in the background, and Google Chrome more or less "open")

1. Tap Explore
2. Tap Trending Repositories
3. Scroll down to sickcodes/Docker-OSX
    ![Screenshot_20230119-080813](https://user-images.githubusercontent.com/2119212/213450803-191d392b-54bb-4955-80d3-2ab886b127c0.png)
4. Tap it
5. Scroll down a bit

## Actual results

* GitHub Mobile app becomes slow/unresponsive 
* Podcasts stops playing (presumably it's killed because the system is starved for memory)

## Expected results

* App should always be responsive
* Background apps shouldn't be harmed

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref thanks for the report. We will look into this. Thanks

**Comment by stevepopovich** [comment]
@jsoref is this still happening for you?

**Reply by jsoref** [reply]
Yes. Although I can't get there via trending.

But searching for `Docker-OSX` and then selecting the repository triggers it.

My podcasts app is killed and eventually github is redrawn to the search results page.
